src/test/solution_space/solution_graph/graph_test.py

```diff
@@ -101,7 +102,11 @@ def create_code_info_chain() -> (List[Tuple[Code, CodeInfo]], List[str]):
 
 def get_vertex_structure(vertex: Vertex) -> VertexStructure:
     source = get_code_from_tree(vertex.serialized_code.canon_tree).strip('\n') if vertex.serialized_code else None
-    return {VERTEX_STRUCTURE.SOURCE: source, VERTEX_STRUCTURE.CODE_INFO_LIST_LEN: len(vertex.code_info_list)}
+    if vertex.serialized_code is None:
```

Comment:
Maybe is it better `code_info_list_len = 0 if vertex.serialized_code is None else sum([len(a_t.code_info_list) for a_t in vertex.serialized_code.anon_trees])`?